app/controllers/draft_events_controller.rb

```diff
@@ -2,14 +2,43 @@ class DraftEventsController < WebsocketRails::BaseController
   def initialize_session
     # perform application setup here
     controller_store[:active_player_index] = 0
-    controller_store[:round_rotation] = [
-      [3, Draft::ROUND_TYPE[0]],
-      [-1, Draft::ROUND_TYPE[1]]
-    ]
-    controller_store[:current_round] = Draft::ROUND_TYPE[0].to_sym
+    controller_store[:round_rotation] = {
+      Draft::ROUND_TYPE[0] => 3,
+      Draft::ROUND_TYPE[1] => -1
+    }
+    controller_store[:round_number] = 1
     controller_store[:players] = User.all.sort
   end
 
+  def current_round_type
+    rotation = controller_store[:round_rotation]
+    if rotation[Draft::ROUND_TYPE[0]] > 0
+      Draft::ROUND_TYPE[0].to_s
+    else
+      Draft::ROUND_TYPE[1].to_s
+    end
+  end
+
+  def next_round
+    rotation = controller_store[:round_rotation]
+    if current_round_type == Draft::ROUND_TYPE[0]
+      rotation[Draft::ROUND_TYPE[0]] -= 1
+      controller_store[:round_number] += 1
+      _message = {
+        round_number: controller_store[:round_number],
+        round_type: Draft::ROUND_TYPE[0]
+      }
+    else
+      controller_store[:round_number] += 1
+      _message = {
+          round_number: controller_store[:round_number],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.